src/components/navigation.js

```diff
@@ -52,12 +53,19 @@ const Navigation = props => {
 	const { logoName, logoNavigateTo } = props
 	const { getTopLevelNavigation, getNonRulesNavigation } = data
 
+	const [isMenuShown, setIsMenuShown] = useState(true)
+	const toggleMenu = () => setIsMenuShown(!isMenuShown)
+
 	return (
-		<aside>
-			{/* className={this.state.showMenu ? 'show' : 'hide'} */}
-			{/* <button className="nav-hide-show-menu" onClick={e => this.handleHideShowMenu()}>
-        ☰
-    </button> */}
+		<aside className={isMenuShown ? 'show' : 'hide'}>
+			{/* hide menu when width <= 600px */}
+			<ReactMedia query="(max-width: 600px)" onChange={matches => matches && isMenuShown && toggleMenu()} />
+			{/* show menu when width > 600px */}
+			<ReactMedia query="(min-width: 601px)" onChange={matches => matches && !isMenuShown && toggleMenu()} />
+			{/* toggle menu  */}
+			<button className="nav-hide-show-menu" onClick={() => toggleMenu()}>
```

Comment:
```suggestion
			<button className="nav-hide-show-menu" onClick=toggleMenu>
```
There should be no need to eta-expand the function.